project/velkoja/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Minimum: create a single static NotificationSent entry. (Just make sure a sane object can be saved)